packages/core/src/serialization/codecs/StylesheetCodec.ts

```
@@ -154,22 +153,21 @@ export class StylesheetCodec extends ObjectCodec {
-              const key = <string>(<Element>entry).getAttribute('as');
+          while (entry) {
+            if (isElement(entry)) {
+              const key = entry.getAttribute('as')!;
 
               if (entry.nodeName === 'add') {
                 const text = getTextContent(<Text>(<unknown>entry));
                 let value = null;
 
-                if (text != null && text.length > 0 && StylesheetCodec.allowEval) {
+                if (text && StylesheetCodec.allowEval) {
                   value = eval(text);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)